linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/rule.yml

```diff
@@ -93,6 +97,9 @@ ocil: |-
 
     If a wireless interface is configured it must be documented and approved by
     the local Authorizing Official.
+    {{% elif 'ubuntu' in product %}}
```

Comment:
this if seems unnecessary as you already has the command introduced above and the note is already in the else below.

Reply:
If we don't make a special conditional for ubuntu it will show the `nmcli` in the `else` statement, which is not ideal since we are not relying on `nmcli`.